tests/test_basics.py

```diff
@@ -1160,5 +1160,5 @@ def recurse():
     # On my machine, it takes about 100-200ms to capture the exception,
     # so this limit should be generous enough.
     assert (
-        capture_end_time - capture_start_time < 10**9
+        capture_end_time - capture_start_time < 20**9
```

Comment:
This is a bit more than 2s :)
```suggestion
        capture_end_time - capture_start_time < 10**9 * 2
```

Reply:
Oh god!